src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -50,9 +52,11 @@
     public static final String EMAIL_DESC_BOB = " " + PREFIX_EMAIL + VALID_EMAIL_BOB;
     public static final String ADDRESS_DESC_AMY = " " + PREFIX_ADDRESS + VALID_ADDRESS_AMY;
     public static final String ADDRESS_DESC_BOB = " " + PREFIX_ADDRESS + VALID_ADDRESS_BOB;
+    public static final String CLASSTIMING_DESC = " " + PREFIX_CLASSTIMING + VALID_CLASSTIMING;
     public static final String TAG_DESC_FRIEND = " " + PREFIX_TAG + VALID_TAG_FRIEND;
     public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
 
+
```

Comment:
is this new line neccesary?

Reply:
Oops my bad, must have accidentally added it